Gopkg.lock

```diff
@@ -133,6 +133,14 @@
   pruneopts = "NUT"
   revision = "24818f796faf91cd76ec7bddd72458fbced7a6c1"
 
+[[projects]]
```

Comment:
Where is the change to Gopkg.toml?

Reply:
Is the Gopkg.lock out of sync with the current Gopkg.toml?

Reply:
I did not make any changes to `Gopkg.toml`.  I did run `dep ensure -add github.com/openshift/cluster-version-operator/pkg/generated/clientset/versioned` to get the client set.  `dep check` reports no problems after this commit.

Reply:
Is this just a consequence of pruning? The constraint is the same, but the subset of files from that constraint preserved in `vendor` will change as a function of packages actually imported by code in our project.

Reply:
We expect Gopkg.lock to be consistently generated (except for pkgs that are not pinned) from Gopkg.toml
Can you check:
```
$ mv Gopkg.lock Gopkg.lock.old
$ dep ensure
$ diff Gopkg.lock Gopkg.lock.old
# This should not show pkg differences 
```

Reply:
@pravisankar, it fails even to generate a new `Gopkg.lock`:
```
% mv Gopkg.lock Gopkg.lock.old
% dep ensure
Solving failure: No versions of github.com/operator-framework/operator-sdk met constraints:
        v0.1.1: Could not introduce github.com/operator-framework/operator-sdk@v0.1.1, as it is not allowed by constraint master from project github.com/openshift/cluster-ingress-operator.
        v0.1.0: Could not introduce github.com/operator-framework/operator-sdk@v0.1.0, as it is not allowed by constraint master from project github.com/openshift/cluster-ingress-operator.
        v0.0.7: Could not introduce github.com/operator-framework/operator-sdk@v0.0.7, as it is not allowed by constraint master from project github.com/openshift/cluster-ingress-operator.
        v0.0.6: Could not introduce github.com/operator-framework/operator-sdk@v0.0.6, as it is not allowed by constraint master from project github.com/openshift/cluster-ingress-operator.
        v0.0.5: Could not introduce github.com/operator-framework/operator-sdk@v0.0.5, as it is not allowed by constraint master from project github.com/openshift/cluster-ingress-operator.
        v0.0.4: Could not introduce github.com/operator-framework/operator-sdk@v0.0.4, as it is not allowed by constraint master from project github.com/openshift/cluster-ingress-operator.
        v0.0.3: Could not introduce github.com/operator-framework/operator-sdk@v0.0.3, as it is not allowed by constraint master from project github.com/openshift/cluster-ingress-operator.
        v0.0.2: Could not introduce github.com/operator-framework/operator-sdk@v0.0.2, as it is not allowed by constraint master from project github.com/openshift/cluster-ingress-operator.
        v0.0.1: Could not introduce github.com/operator-framework/operator-sdk@v0.0.1, as it is not allowed by constraint master from project github.com/openshift/cluster-ingress-operator.
        master: Could not introduce github.com/operator-framework/operator-sdk@master due to multiple problematic subpackages:
        Subpackage github.com/operator-framework/operator-sdk/pkg/k8sclient is missing. (Package is required by (root).)        Subpackage github.com/operator-framework/operator-sdk/pkg/util/k8sutil is missing. (Package is required by (root).)
        revert-712-decompose_gopath: Could not introduce github.com/operator-framework/operator-sdk@revert-712-decompose_gopath, as it is not allowed by constraint master from project github.com/openshift/cluster-ingress-operator.
        v0.0.7: Could not introduce github.com/operator-framework/operator-sdk@v0.0.7, as it is not allowed by constraint master from project github.com/openshift/cluster-ingress-operator.
        v0.1.x: Could not introduce github.com/operator-framework/operator-sdk@v0.1.x, as it is not allowed by constraint master from project github.com/openshift/cluster-ingress-operator.
zsh: exit 1     dep ensure
dep ensure  8.23s user 7.95s system 78% cpu 20.567 total
%
```